it/v3/endpoints/CreateAndAmendOtherExpensesControllerISpec.scala

```diff
@@ -47,8 +46,7 @@ class CreateAndAmendOtherExpensesControllerISpec extends IntegrationBaseSpec {
         }
 
         val response: WSResponse = await(request().put(requestBodyJson))
-        response.status shouldBe OK
-        response.json shouldBe responseBody
```

Comment:
this can be replaced to verify that there is no body
response.body.isEmpty shouldBe true